docs/api/covidcast-signals/nchs-mortality.md

```diff
@@ -0,0 +1,56 @@
+---
+title: NCHS Mortality Data
+parent: Data Sources and Signals
+grand_parent: COVIDcast API
+---
+
+# NCHS Mortality Data
+{: .no_toc}
+
+* **Source name:** ``
+* **Number of data revisions since 19 May 2020:** 0
+* **Date of last change:** Never
+* **Available for:** state (see [geography coding docs](../covidcast_geography.md))
+
+This data source of Provisional Death Counts for COVID-19 is based on reports made
+available by National Center for Health statistics [(NCHS)](https://www.cdc.gov/nchs/nvss/vsrr/COVID19/index.htm).
+
+| Signal | Description |
+| --- | --- |
+| `covid_deaths_num` | Number of all the weekly new deaths with confirmed or presumed COVID-19 |
+| `covid_deaths_prop` | Number of all the weekly new deaths with confirmed or presumed COVID-19 per 100,000 population |
+| `total_deaths_num` | Number of weekly new deaths from all causes |
+| `total_deaths_prop` | Number of weekly new deaths from all causes per 100,000 population |
+| `influenza_deaths_num` | Number of weekly new deaths involving Influenza |
+| `influenza_deaths_prop` | Number of weekly new deaths involving Influenza per 100,000 population |
+| `pneumonia_deaths_num` | Number of weekly new deaths involving Pneumonia, excluding Influenza deaths |
+| `pneumonia_deaths_prop` | Number of weekly new deaths involving Pneumonia, excluding Influenza deaths per 100,000 population |
+| `pneumonia_and_covid_deaths_num`| Number of weekly new deaths involving COVID-19 and Pneumonia, excluding Influenza |
+|`pneumonia_and_covid_deaths_prop`| Number of weekly new deaths involving COVID-19 and Pneumonia, excluding Influenza per 100,000 population |
+|`pneumonia_influenza_or_covid_19_deaths_num`| Number of weekly new deaths involving Pneumonia, Influenza, or COVID-19|
+|`pneumonia_influenza_or_covid_19_deaths_prop`| Number of weekly new deaths involving Pneumonia, Influenza, or COVID-19 per 100,000 population|
+|`percent_of_expected_deaths`| The number of weekly new deaths for all causes in 2020 compared to the average number across the same week in 2017–2019|
```

Comment:
This is vague; how is it compared, and what is the percent of?

Reply:
I believe it's literally

` 100 * (number of weekly new deaths for all causes in 2020) / (average number of weekly new deaths for all causes across the same week in 2017–2019)`